packages/truffle-debugger/lib/data/selectors/index.js

```diff
@@ -662,6 +727,17 @@ const data = createSelectorTree({
             Object.assign({}, ...keyedResults)
           );
         }
+      ),
+
+      /*
+       * data.current.identifiers.thisDefinition
+       *
+       * returns a spoofed definition for the this variable
+       */
+      thisDefinition: createLeaf(
```

Comment:
Hm, I guess I agree that putting it underneath `data.current.identifiers.definitions` makes sense.  I think I'd still call it `thisDefinition` though.  I'd like to avoid the `this` name except where there's a good reason for it.

Reply:
Oh except I guess in this case it is literally the `this` member of `data.current.identifiers.definitions`.  Yeah, OK, `data.current.identifiers.definitions.this` it is.